### DISCUSSION TITLE
Welcome to LCUI Discussions!

### DISCUSSION THREAD

**Discussion by lc-soft** [lc-soft/LCUI#229] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by drsounds** [comment]
Hi, 

I am a developer from Sweden which find this project exciting because it is new alternative from de facto standard solutions on the market such CEF, Electron etc. and remainds me on an internal GUI framework the Swedish startup Spotify created in the past but also that it is an open Chinese innovation that shows China can export good stuff to the society
